mavis/validate/main.py

```diff
@@ -26,11 +26,11 @@
 
 
 def main(
-    input, output,
+    inputs, output,
     bam_file, strand_specific,
     library, protocol, median_fragment_size, stdev_fragment_size, read_length,
     reference_genome, reference_genome_filename, annotations, masking, aligner_reference,
-    samtools_version, start_time=int(time.time()), **kwargs
+    samtools_version, start_time=int(time.time()), filename_prefix='validate', **kwargs
 ):
     """
     Args:
```

Comment:
inputs is now a list right? should update the docstring here

Reply:
I have updated this in commit d03ecabce02ea3e21671e5b74df4c8f84f16993f